src/Symfony/Component/HttpKernel/DataCollector/LoggerDataCollector.php

```diff
@@ -87,6 +103,35 @@ public function countScreams()
         return isset($this->data['scream_count']) ? $this->data['scream_count'] : 0;
     }
 
+    public function getAppBootstrappingLogs()
+    {
+        return isset($this->data['app_bootstrapping_logs']) ? $this->data['app_bootstrapping_logs'] : array();
+    }
+
+    public function countAppBootstrapping()
+    {
+        return isset($this->data['app_bootstrapping_count']) ? $this->data['app_bootstrapping_count'] : 0;
+    }
+
+    public function getCompilerLogs()
+    {
+        return isset($this->data['compiler_logs']) ? $this->data['compiler_logs'] : array();
+    }
+
+    private function getBootLogsFile()
+    {
+        $file = current(glob($this->cacheDir.'/*DebugProjectContainerBootLogs.log'));
```

Comment:
this should not be hardcoded but should replicate the way the name is built - maybe just pass the prefix as argumetn instead of only the cache_dir ? (there is parameter that holds the name of the container class)